django_netjsongraph/base/link.py

```diff
@@ -37,9 +38,21 @@ class AbstractLink(TimeStampedEditableModel):
                            load_kwargs={'object_pairs_hook': OrderedDict},
                            dump_kwargs={'indent': 4})
 
+    link_status_change = Signal(providing_args=["topology", "id"])
+
     class Meta:
         abstract = True
 
+    def __init__(self, *args, **kwargs):
+        super(AbstractLink, self).__init__(*args, **kwargs)
+        self.initial_status = self.status
+
+    def save(self, *args, **kwargs):
+        if self.status != self.initial_status:
```

Comment:
move it after the `save()` call, so the signal is sent after the status has changed